completed javascript-loops-practice - YoonHwan Cho

YoonHCho opened the pull request:
completed javascript-loops-practice - YoonHwan Cho
![2022-07-08 14_44_58-Practice Problems - 4](https://user-images.githubusercontent.com/106636383/178074542-53f0e6e6-2dac-461d-8a2c-7f91196266a8.png)

codingcodymiller commented on javascript-loops-practice/filter-out-strings.js:
```diff
@@ -0,0 +1,10 @@
+function filterOutStrings(values) {
+  var array = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```
This function is meant to filter out strings, not gather all numbers. What if the input included other data types?

codingcodymiller commented on javascript-loops-practice/odd-or-even.js:
```diff
@@ -0,0 +1,10 @@
+  for (var i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      newArray.push('even');
+    } else newArray.push('odd');
```
Keep those curly braces, please 🙏

YoonHCho commented:
thanks, didn't catch about what if other data types. changed the condition to not equal to string. and added the curly braces with else of evenorodd function
![2022-07-08 16_24_22-filter-out-strings js - c0622-code-solutions  Dev Container  - Visual Studio Cod](https://user-images.githubusercontent.com/106636383/178081811-e6a358ff-bfd9-4074-b0fe-42bb59c74791.png)
![2022-07-08 16_25_20-odd-or-even js - c0622-code-solutions  Dev Container  - Visual Studio Code](https://user-images.githubusercontent.com/106636383/178081813-091e4717-5a26-4e53-9c66-b580ea404523.png)
